code/Attack/DrupalScan.py

```diff
@@ -126,102 +140,132 @@ def generate_attack_packets(self):
-                            break
-                    destination_origin_wins[destination_origin_win] = destination_win_rand_pick
-                new_win = destination_origin_wins[destination_origin_win]
-                tcp_pkt.setfieldval("window", new_win)
-                # MSS
-                tcp_options = tcp_pkt.getfieldval("options")
-                if tcp_options:
-                    if tcp_options[0][0] == "MSS":
-                        tcp_options[0] = ("MSS", mss_value)
-                        tcp_pkt.setfieldval("options", tcp_options)
 
+                # We do not affect the window size & MSS values for internal traffic to DB
+                if ip_pkt.getfieldval("dst") == self.MYSQL_DB_IP:
+                    # set dst values (victim -> db)
+                    eth_frame.setfieldval("dst", mac_sql)         
+                    ip_pkt.setfieldval("dst", ip_sql)
+                    tcp_pkt.setfieldval("sport", ep)  # this line has no effect - only for clarification
+                    tcp_pkt.setfieldval("dport", self.MY_SQL_PORT)  # generated source port value for convo 
+                    
+
+                else:# (victim -> attack)
+                    # set dst values (victim -> attack)
+                    eth_frame.setfieldval("dst", mac_source)
+                    ip_pkt.setfieldval("dst", ip_source)
+                    ip_pkt.setfieldval("ttl", destination_ttl_value)
+                    tcp_pkt.setfieldval("sport",  self.HTTP_PORT) #  this line has no effect only for clarification
+                    tcp_pkt.setfieldval("dport", ep)
+                    
+                    # Window Size (mapping)
+                    destination_origin_win = tcp_pkt.getfieldval("window")
+                    if destination_origin_win not in destination_origin_wins:
+                        while True:
+                            destination_win_rand_pick = destination_win_prob_dict.random()
+                            if destination_win_rand_pick != 0:
+                                break
+                        destination_origin_wins[destination_origin_win] = destination_win_rand_pick
+                    new_win = destination_origin_wins[destination_origin_win]
+                    tcp_pkt.setfieldval("window", new_win)
+                    # MSS
+                    tcp_options = tcp_pkt.getfieldval("options")
+                    if tcp_options:
+                        if tcp_options[0][0] == "MSS":
+                            tcp_options[0] = ("MSS", mss_value)
+                            tcp_pkt.setfieldval("options", tcp_options)
+                
+                # Generate the packet
                 new_pkt = (eth_frame / ip_pkt / tcp_pkt)
-                timestamp_next_pkt = self.timestamp_controller.next_timestamp() + inter_arrival_times[self.pkt_num]
+                timestamp_next_pkt = self.timestamp_controller.next_timestamp() #+# inter_arrival_times[self.pkt_num]
```

Comment:
Please remove out commented code.

If this is the final version of the attack, it should not be necessary to include this.